ledger/participant-integration-api/src/test/suite/scala/platform/store/CompletionFromTransactionSpec.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store
+
+import com.daml.ledger.offset.Offset
+import com.google.protobuf.timestamp.Timestamp
+import com.google.rpc.status.Status
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.time.Instant
+
+class CompletionFromTransactionSpec extends AnyWordSpec with Matchers {
```

Comment:
Could you also check the period field then?

Reply:
Done